tests/Feature/BookTest.php

```diff
@@ -92,6 +98,25 @@ public function test_get_all_unclaimed_books(): void
 
     }
 
+    public function test_get_all_claimed_books_invalid(): void
+    {
+        Book::factory()->create();
+        $book = Book::factory()->create();
+
+        $book->id = 2;
+        $book->claimed_by_name = '';
+
+      
+        $book->save();
```

Comment:
This is very minor - but since you're just testing the validation you can actually remove these lines. There don't need to be any books in the system in order to test the validation.

It is worth taking the time to remove this as creating test data you don't need will slow the test suite down. Won't make any difference at all at this point, but if this project grew to the point where you had many hundreds of tests, small savings like this could save a decent bit of time each time you run the tests

Reply:
Hi Ash, thank you for this comment.  We have removed these un-required lines of code.